lib/haml_coffee_assets/action_view/resolver.rb

```diff
@@ -8,7 +8,7 @@ module ActionView
     # in it would normally be a fallback for all formats.
     #
     class Resolver < ::ActionView::FileSystemResolver
-      def find_templates(name, prefix, partial, details)
+      def find_templates(name, prefix, partial, details, outside_app_allowed = false)
```

Comment:
You can have both defined arguments and a splat argument in the same method argument list, i.e., `def find_templates(name, prefix, partial, details, *)`. Any additional arguments are gobbled up by the `*` if they're present, and when calling `super` they're passed through. Since there's no default value, there's no arity issue with previous versions of Rails.